pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -49,46 +49,46 @@
 
 platform = 'gnss'
 name = 'tec'
-tags = {'vtec': 'vertical TEC'}
+tags = {'vtec': 'vertical TEC', 'site': 'Sites used in daily TEC data'}
 inst_ids = {'': [tag for tag in tags.keys()]}
 
 pandas_format = False
 
 # Local attributes
-dname = '{{year:02d}}{{month:02d}}{{day:02d}}'
+dname = '{{month:02d}}{{day:02d}}'
 vname = '.{{version:03d}}'
-supported_tags = {ss: {'vtec': ''.join(['gps', dname, 'g', vname,
+supported_tags = {ss: {'vtec': ''.join(['gps{{year:02d}}', dname, 'g', vname,
+                                        ".{file_type}"]),
+                       'site': ''.join(['site_{{year:04d}}', dname, vname,
                                         ".{file_type}"])}
                   for ss in inst_ids.keys()}
 remote_tags = {ss: {kk: supported_tags[ss][kk].format(file_type='hdf5')
```

Comment:
Tests have an error that I think is related to the difference in file extension, '.h5' vs '.hdf5'
```
files = ['site_20010101.001.h5']
format_str = 'site_{year:04d}{month:02d}{day:02d}.{version:03d}.hdf5'
```

Reply:
This was also an issue in the general instrument pull.  Added a fix in the general 'list_remote_files' function.